### ISSUE TITLE
Bug report: uploading sklearn-onnx models

### ISSUE THREAD

**Issue by nlathia** [operatorai/modelstore#156]
🐛 A bug has been reported to me, where `modelstore` does not find a matching manager when trying to upload an `onnx` model. This needs to be investigated to see if we can replicate it, and then fix it.

In this specific instance, the `type` of the model is:
```
<class 'onnx.onnx_ml_pb2.ModelProto'>
```

Which looks like it's different from the `type` that `modelstore` checks against.

The dependencies that were being used by the reporter are `modelstore>=0.0.73` with:

```
google-cloud-bigquery==2.28.0
google-cloud-core==2.1.0
imblearn==0.0
jinja2==2.11.3
lightgbm==3.2.1
onnx==1.7.0
onnxconverter-common==1.7.0
onnxmltools==1.7.0
onnxruntime==1.6.0
optuna==2.8.0
pandas==1.1.4
pyarrow==4.0.1
pytest==3.6.3
scikit-learn==0.24.1
skl2onnx==1.7.0
tqdm==4.54.1
```

And a report that:

> skl2onnx is the problem, anything above 1.7.0 will cause that manager not found error

**Comment by nlathia** [comment]
I've added a specific example for this case here in the PR above, as:

```
examples/examples-by-ml-library/libraries/onnx_lightgbm_example.py
```

And I've also:

- Upgraded all of the dependencies in the example virtual environment;
- Relaxed the eligibility for Onnx (before: it required both `onnx` and `onnxruntime` to be installed; now it just needs `onnx`).

Right now I'm struggling to further replicate this bug 🙏 so will reach out to the reporter for more details

**Comment by nlathia** [comment]
This is fixed in `modelstore==0.0.74`
